docs/resources/aws_security_group.md.erb

```diff
@@ -91,6 +90,26 @@ The following examples show how to use this InSpec audit resource.
       its('outbound_rules.count') { should cmp 3 }
     end
 
+    # Ensure that one of the security groups in a list allows access from one of the groups in another list
+    # In this example we get the list of groups from Terraform output
+    # load data from Terraform output
+    content = inspec.profile.file("terraform.json")
```

Comment:
I think this is a powerful example, but will also be intimidating to new users.  We also don't want to imply that you must use Terraform to use this feature.  I'd recommend replacing this with a simplified example.

Reply:
I had some doubt about this already, I chose to put it in because it took me a while to  figure out how to do this properly. This is an actual use case for us and I do think it might be useful. Would it be acceptable if I remove the reference to terraform, and add a less complex example?

Reply:
@clintoncwolfe PTAL